public/landing.html

```diff
@@ -28,6 +28,282 @@
+            </div>
+
+            <div class="checkbox">
+              <label>
+                <input type="checkbox"> I agree to <a href="#">Filebox terms</a>
+              </label>
+            </div>
+
+            <div class="form-group">
+              <button type="submit" class="btn btn-primary btn-block">Sign up for free</button>
+            </div>
+
+            <hr>
+
+            <div class="form-group">
+              <button type="submit" class="btn btn-primary btn-block"><i class="fa fa-google"></i> Sign up free with Google</button>
+            </div>
+          </form>
+        </div>
+      </div>
+
+      <div class="row padding-inside-top-lg"></div>
+      <div class="col-md-offset-1 col-md-4">
+        <h1>Take your docs anywhere</h1>
+
+        <p>
+          Save files on your computer, then access them on your phone from the road. Everything you keep in Filebox is synced automatically to all your devices.
+        </p>
+      </div>
+      <div class="col-md-offset-2 col-md-4">
+        <img src="assets/images/docs-anywhere.png" alt="">
+      </div>
+    </div>
+
+    <div class="row padding-inside-top-lg">
+      <div class="col-md-offset-1 col-md-4">
+        <img src="assets/images/send-videos-quickly.png" alt="">
+      </div>
+
+      <div class="col-md-offset-2 col-md-4">
+        <h1>Send videos quickly</h1>
+
+        <p>
+          Send your entire wedding video to family with a simple link. It’s easy to share large files with anyone — even if they don’t have a Filebox account.
+        </p>
+      </div>
+    </div>
+
+    <div class="row padding-inside-top-lg"></div>
+    <div class="col-md-offset-1 col-md-4">
+      <h1>Keep your photos safe</h1>
+
+      <p>
+        Back up vacation photos automatically from your phone or computer. That way, memories are safe as soon as you make them, and you can relive them from any device.
+      </p>
+    </div>
+    <div class="col-md-offset-2 col-md-4">
+      <img src="assets/images/keep-your-photos-safe.png" alt="">
+    </div>
+  </div>
+
+  <div class="row padding-inside-top-lg"></div>
+  <div class="col-md-offset-1 col-md-4">
+    <img src="assets/images/work-on-slides-together.png" alt="">
+  </div>
+
+  <div class="col-md-offset-2 col-md-4">
+    <h1>Work on slides together</h1>
+
+    <p>
+      Edit a presentation with teammates without emailing files back and forth. When you edit a file in a shared folder, everyone gets the update automatically.
+    </p>
+  </div>
+</div>
+
+<div class="row padding-inside-top-lg"></div>
+
+<div class="col-md-offset-1 col-md-4">
+  <h1>Never lose a file again</h1>
+
+  <p>
+    Left your phone on the train? Your photos, docs, and videos are safe. Just sign in to Filebox from any device, and your files will be there waiting for you.
+  </p>
+</div>
+
+<div class="col-md-offset-2 col-md-4">
+  <img src="assets/images/never-lose-a-file-again.png" alt="">
+</div>
+</div>
+
+<div class="row padding-inside-top-lg">
```

Comment:
What is the goal of this row? Just to create whitespace? If so, maybe just increase the whitespace below/above the surrounding rows.